client/src/pages/SignUp/SignUp.tsx

```diff
@@ -38,24 +39,26 @@ export default function Register(): JSX.Element {
   };
 
   return (
-    <Grid container component="main" className={classes.root}>
-      <CssBaseline />
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
-        <Box className={classes.authWrapper}>
-          <AuthHeader linkTo="/login" asideText="Already have an account?" btnText="Login" />
-          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
-            <Grid container>
-              <Grid item xs>
-                <Typography className={classes.welcome} component="h1" variant="h5">
-                  Create an account
-                </Typography>
+    <Dialog open fullWidth maxWidth="sm">
+      <Grid container component="main" className={classes.root}>
+        <CssBaseline />
+        <Grid item xs={12} sm={12} md={12} lg={12} elevation={6} component={Paper} square>
+          <Box className={classes.authWrapper}>
+            <Box width="65%" maxWidth={600} p={2} alignSelf="center">
```

Comment:
Ditto here

Reply:
Changed the element's inline styling to useStyle sheet.